pkg/exec/exec.go

```diff
@@ -132,11 +142,29 @@ func execCmdInPod(ctx context.Context, clientsets *k8sutil.Clientsets, command,
 		logging.Fatal(err)
 	}
 
-	// Connect this process' std{in,out,err} to the remote shell process.
-	return exec.StreamWithContext(ctx, remotecommand.StreamOptions{
-		Stdin:  os.Stdin,
-		Stdout: stdout,
-		Stderr: stderr,
-		Tty:    false,
-	})
+	// returnOutput is true, the command's output will be print on shell directly with os.Stdout or os.Stderr
+	if !returnOutput {
+		// Connect this process' std{in,out,err} to the remote shell process.
+		err = exec.StreamWithContext(ctx, remotecommand.StreamOptions{
+			Stdin:  os.Stdin,
+			Stdout: os.Stdout,
+			Stderr: os.Stderr,
+			Tty:    false,
+		})
+		if err != nil {
+			logging.Fatal(err)
```

Comment:
Why is this one fatal, but line 167 is not? Seems like we either need the `exitOnError` or I need to understand better how it is tied to the `returnOutput`

Reply:
oh that will overlook, we need to use `logging.Fatal` both places since if we are not able to create the stream to capture or print the output then we want to exit right there.